src/core/Akka/Configuration/Hocon/HoconObject.cs

```diff
@@ -161,6 +161,7 @@ public string ToString(int indent)
             var sb = new StringBuilder();
             foreach (var kvp in Items)
             {
+                if (kvp.Value.AdoptedFromFallback) continue;
```

Comment:
Exactly, either way logs get bloated, even some tests become failing; First I tried to use subclass of HoconValue as a marker, but this somehow breaks tests e.g LoadPluginSpec. Couple of hours of debugging yield no result for me, I suspect the system can't instantiate correct dispatcher for specs. But how inheritance affect this remains mystery for me.